### PR TITLE
-create Service, Storage, Application

### PR DESCRIPTION
**Pull request by Burito2021**
-add Faker library
-correct gradle dependencies, gradle java version
-add unit tests
-move methods from Storage to Service class
-add gradle.properties file

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -1,21 +1,32 @@
 plugins {
     id 'java'
 }
```
Comment (MaximDzhezhelo) [inline build.gradle:4]: Make offset

File: build.gradle [hunk 2]
```diff
@@ -1,21 +1,32 @@
...
-    testRuntimeOnly 'org.junit.jupiter:junit-jupiter-engine:5.7.2'
+    implementation("org.junit.jupiter:junit-jupiter-api:$junit")
+    implementation("com.github.javafaker:javafaker:$faker")
+    testImplementation ("org.hamcrest:hamcrest-core:$hamcrestcore")
+    testImplementation ("org.hamcrest:hamcrest:$hamcrest")
+    testImplementation ("org.hamcrest:hamcrest-junit:$hamcrestjunit")
+    testImplementation ("org.assertj:assertj-core:$assertj")
+    implementation("org.junit.jupiter:junit-jupiter-params:$junit")
+    testRuntimeOnly("org.junit.jupiter:junit-jupiter-engine:$junit")
+    testRuntimeOnly("com.github.javafaker:javafaker:$faker")
```
Comment (MaximDzhezhelo) [inline build.gradle]: Remove offset

File: settings.gradle [hunk 3]
```diff
@@ -1,2 +1,2 @@
-rootProject.name = 'interface And Abstract'
+rootProject.name = 'interface_And_Abstract'
```
Comment (MaximDzhezhelo) [inline settings.gradle]: Don't use CamelSpace.
https://www.oracle.com/java/technologies/javase/codeconventions-namingconventions.html
https://www.oracle.com/java/technologies/javase/codeconventions-filenames.html

File: src/main/java/net/education/kyivstar/Application.java [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+
+import java.util.Random;
+
+import static net.education.kyivstar.Service.UserType.*;
+import static net.education.kyivstar.Storage.*;
+
+public class Application {
+
+    public static void main(String[] args) {
+        Faker faker = new Faker();
```
Comment (MaximDzhezhelo) [inline src/main/java/net/education/kyivstar/Application.java]: Use var keyword

File: src/main/java/net/education/kyivstar/Application.java [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+        Random random = new Random();
+
+        Service service = new Service(faker,storage,random);
+
+        service.populateStorage(1000);
+
+        service.createUserAndStore(STUDENT,"Alex","Burmistrenko",30);
+        service.createUserAndStore(TEACHER,"Maxim","Dzhezhelo",30);
+
+        System.out.println("Get by name method: " + service.getByName("Alex"));
```
Comment (MaximDzhezhelo) [inline src/main/java/net/education/kyivstar/Application.java:24]: Use Logger
https://betterstack.com/community/guides/logging/how-to-start-logging-with-java/

**Review by MaximDzhezhelo — CHANGES_REQUESTED** [review]

File: settings.gradle (hunk 3 above)
Comment (Burito2021) [reply settings.gradle]: According to the convention**Such conventions might specify that certain directory name components be division, department, project, machine, or login names.**  **example: edu.cmu.cs.bovik.cheese**,Then it is to call the neme of the project  **rootProject.name = 'net.education'**, Yes?

File: build.gradle (hunk 1 above)
Comment (Burito2021) [reply build.gradle:4]: done

File: build.gradle (hunk 2 above)
Comment (Burito2021) [reply build.gradle]: done

File: settings.gradle (hunk 3 above)
Comment (Burito2021) [reply settings.gradle]: done

File: src/main/java/net/education/kyivstar/Application.java (hunk 4 above)
Comment (Burito2021) [reply src/main/java/net/education/kyivstar/Application.java]: done

File: src/main/java/net/education/kyivstar/Application.java (hunk 5 above)
Comment (Burito2021) [reply src/main/java/net/education/kyivstar/Application.java:24]: done

Comment (MaximDzhezhelo) [reply src/main/java/net/education/kyivstar/Application.java:24]: Why didn't you remove sout? 

please, follow this https://gist.github.com/robertpainsi/b632364184e70900af4ab688decf6f53

**Review by MaximDzhezhelo — CHANGES_REQUESTED** [review]

**Pull request closed without merging**